src/ListActions/ListActions.less

```diff
@@ -1,115 +1,54 @@
 /* List Actions */
 .moon-list-actions {
-	position:relative;
-	display:inline-block;
-	overflow:visible;
-
-	.moon-icon-button {
-		margin:0;
+	.moon-hspacing > :first-child& .moon-icon,
+	.moon-hspacing > :first-child & .moon-icon {
+		margin-left: 0;
+
+		.enyo-locale-right-to-left & {
+			margin-left: @moon-spotlight-outset;
+			margin-right: 0;
+		}
 	}
-}
-.list-actions-drawer {
-	position: absolute;
-	top: 0;
-	left: 0;
-	bottom: 0;
-	right: 0;
-}
-.moon-header .list-actions-drawer {
-	top: -@moon-header-border-top-width;
-	bottom: -@moon-header-border-bottom-width;
-}
 
-/* Close button */
-.moon-icon-button.moon-list-actions-close {
-	position: absolute;
-	right: @moon-spotlight-outset;
-	top: @moon-spotlight-outset;
-	z-index: 2;
-}
-.enyo-locale-right-to-left .moon-icon-button.moon-list-actions-close {
-	right: auto;
-	left: @moon-spotlight-outset;
-}
+	.moon-hspacing > :last-child& .moon-icon,
+	.moon-hspacing > :last-child & .moon-icon {
+		margin-right: 0;
 
-/* Scroller */
-.moon-list-actions-scroller {
-	margin: 18px;
-	margin-right: @moon-icon-button-small-size + 18px;
-	padding: 0px;
-	z-index: 1;
-}
-.enyo-locale-right-to-left .moon-list-actions-scroller {
-	margin-right: 12px;
-	margin-left: @moon-icon-button-small-size + 18px;
-}
-
-/* Action menu */
-.moon-list-actions-menu {
-	display: inline-block;
-	vertical-align: top;
-	width: 300px;		/* Do not change - used in JS */
-	min-width: 300px;	/* Do not change - used in JS */
-	float: right;
-	box-sizing: border-box;
-}
-.enyo-locale-right-to-left .moon-list-actions-menu {
-	float: left;
+		.enyo-locale-right-to-left & {
+			margin-left: 0;
+			margin-right: @moon-spotlight-outset;
+		}
+	}
 }
 
-.moon-list-actions-drawer.proportional-width.stacked .moon-list-actions-menu {
-	width: 100% !important;
-}
-.moon-list-actions-drawer.stacked .moon-list-actions-menu {
-	margin-bottom: 18px;
-	clear: both;
-}
-.moon-list-actions-menu .enyo-scroller {
-	max-width: 100%;
+.list-actions-hunits(@property, @n) {
+	&:before,
+	&:after {
+		@{property}: (@moon-icon-button-small-size * (@n - 1)) + (@moon-icon-button-small-size * @n) / 2 - @moon-icon-margin * @n / 2;
+	}
 }
 
-/* Drawer */
-.moon-list-actions-drawer {
-	overflow: hidden;
-	z-index: 50;
-	pointer-events: none;
-	* {
-		pointer-events: auto;
+.generate-columns(@n, @i: 1) when (@i =< @n) {
```

Comment:
Suggest moving this within `.moon-list-actions-popup`. As it stands, it'd a global LESS function and could be stomped by LESS elsewhere. Since it's truly private to this control, we should scope it as such.